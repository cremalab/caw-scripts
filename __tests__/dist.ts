import spawn = require("cross-spawn")

const removeFirstLine = (a: string) =>
  a
    .split("\n")
    .slice(2)
    .join("\n")

describe("dist", () => {
  it("works", () => {
    spawn.sync("npm", ["run", "build"])
    process.chdir("./dist")
    const tree = spawn.sync("treee", ["-a", "-l 6"]).output.toString()
    expect(removeFirstLine(tree)).toMatchSnapshot()
  })
})
