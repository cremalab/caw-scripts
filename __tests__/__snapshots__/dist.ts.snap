// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dist works 1`] = `
"â”œâ”€â”€ .storybook
|  â”œâ”€â”€ addons.d.ts
|  â”œâ”€â”€ addons.js
|  â”œâ”€â”€ addons.js.map
|  â”œâ”€â”€ config.d.ts
|  â”œâ”€â”€ config.js
|  â”œâ”€â”€ config.js.map
|  â”œâ”€â”€ webpack.config.d.ts
|  â”œâ”€â”€ webpack.config.js
|  â””â”€â”€ webpack.config.js.map
â”œâ”€â”€ index.d.ts
â”œâ”€â”€ index.js
â”œâ”€â”€ index.js.map
â”œâ”€â”€ scripts
|  â”œâ”€â”€ build.d.ts
|  â”œâ”€â”€ build.js
|  â”œâ”€â”€ build.js.map
|  â”œâ”€â”€ build:visual.d.ts
|  â”œâ”€â”€ build:visual.js
|  â”œâ”€â”€ build:visual.js.map
|  â”œâ”€â”€ init.d.ts
|  â”œâ”€â”€ init.js
|  â”œâ”€â”€ init.js.map
|  â”œâ”€â”€ new:component.d.ts
|  â”œâ”€â”€ new:component.js
|  â”œâ”€â”€ new:component.js.map
|  â”œâ”€â”€ new:type.d.ts
|  â”œâ”€â”€ new:type.js
|  â”œâ”€â”€ new:type.js.map
|  â”œâ”€â”€ new:util.d.ts
|  â”œâ”€â”€ new:util.js
|  â”œâ”€â”€ new:util.js.map
|  â”œâ”€â”€ start.d.ts
|  â”œâ”€â”€ start.js
|  â”œâ”€â”€ start.js.map
|  â”œâ”€â”€ start:ci.d.ts
|  â”œâ”€â”€ start:ci.js
|  â”œâ”€â”€ start:ci.js.map
|  â”œâ”€â”€ start:visual.d.ts
|  â”œâ”€â”€ start:visual.js
|  â”œâ”€â”€ start:visual.js.map
|  â”œâ”€â”€ start:visual:ci.d.ts
|  â”œâ”€â”€ start:visual:ci.js
|  â”œâ”€â”€ start:visual:ci.js.map
|  â”œâ”€â”€ test:lint.d.ts
|  â”œâ”€â”€ test:lint.js
|  â”œâ”€â”€ test:lint.js.map
|  â”œâ”€â”€ test:lint:fix.d.ts
|  â”œâ”€â”€ test:lint:fix.js
|  â”œâ”€â”€ test:lint:fix.js.map
|  â”œâ”€â”€ test:unit.d.ts
|  â”œâ”€â”€ test:unit.js
|  â”œâ”€â”€ test:unit.js.map
|  â”œâ”€â”€ test:unit:coverage.d.ts
|  â”œâ”€â”€ test:unit:coverage.js
|  â”œâ”€â”€ test:unit:coverage.js.map
|  â”œâ”€â”€ test:unit:coverage:ci.d.ts
|  â”œâ”€â”€ test:unit:coverage:ci.js
|  â”œâ”€â”€ test:unit:coverage:ci.js.map
|  â”œâ”€â”€ test:visual.d.ts
|  â”œâ”€â”€ test:visual.js
|  â”œâ”€â”€ test:visual.js.map
|  â”œâ”€â”€ test:visual:approve.d.ts
|  â”œâ”€â”€ test:visual:approve.js
|  â”œâ”€â”€ test:visual:approve.js.map
|  â”œâ”€â”€ test:visual:ci.d.ts
|  â”œâ”€â”€ test:visual:ci.js
|  â”œâ”€â”€ test:visual:ci.js.map
|  â”œâ”€â”€ test:visual:update.d.ts
|  â”œâ”€â”€ test:visual:update.js
|  â””â”€â”€ test:visual:update.js.map
â”œâ”€â”€ types.d.ts
â”œâ”€â”€ types.js
â”œâ”€â”€ types.js.map
â””â”€â”€ utils
   â”œâ”€â”€ answerPrompts.d.ts
   â”œâ”€â”€ answerPrompts.js
   â”œâ”€â”€ answerPrompts.js.map
   â”œâ”€â”€ args.d.ts
   â”œâ”€â”€ args.js
   â”œâ”€â”€ args.js.map
   â”œâ”€â”€ paths.d.ts
   â”œâ”€â”€ paths.js
   â”œâ”€â”€ paths.js.map
   â”œâ”€â”€ removeReactAppEnv.d.ts
   â”œâ”€â”€ removeReactAppEnv.js
   â”œâ”€â”€ removeReactAppEnv.js.map
   â”œâ”€â”€ runHygen.d.ts
   â”œâ”€â”€ runHygen.js
   â””â”€â”€ runHygen.js.map

directory: 3 file: 87

ignored


[2K[1G,"
`;
